src/OSPSuite.Core/Domain/Builder/IndividualBuildingBlock.cs

```diff
@@ -0,0 +1,31 @@
+﻿using OSPSuite.Core.Domain.Services;
+
+namespace OSPSuite.Core.Domain.Builder
+{
+   public class IndividualBuildingBlock : PathAndValueEntityBuildingBlockSourcedFromPKSimBuildingBlock<IndividualParameter>
```

Comment:
I wasn't sure if CalculationMethodCache should also be part of the MoBi Individual. These are the common building block properties besides that one.

Reply:
That's a mouthful class name.

Reply:
What about PathAndValueEntityBuildingBlockFromPKSim